src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.AcceptAllCerts.cs

```diff
@@ -0,0 +1,73 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net.Security;
+using System.Net.Test.Common;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Net.Http.Functional.Tests
+{
+    public class HttpClientHandler_DangerousAcceptAllCertificatesValidator_Test
+    {
+        [Fact]
+        public void SingletonReturnsTrue()
+        {
+            Assert.NotNull(HttpClientHandler.DangerousAcceptAnyServerCertificateValidator);
+            Assert.Same(HttpClientHandler.DangerousAcceptAnyServerCertificateValidator, HttpClientHandler.DangerousAcceptAnyServerCertificateValidator);
+            Assert.True(HttpClientHandler.DangerousAcceptAnyServerCertificateValidator(null, null, null, SslPolicyErrors.None));
+        }
+
+        [Theory]
+        [InlineData(SslProtocols.Tls, false)]
```

Comment:
I can cull it back and add a comment.  It was mainly to ensure we're appropriately setting version constraints even when the callback is applied (since I know in the implementation that has to be done in particular places).